docs/source/install.rst

```diff
@@ -68,6 +72,10 @@ UCX
     git clone https://github.com/openucx/ucx
     cd ucx
     git checkout v1.8.x
+    # apply UCX IB registration patches
+    curl -L https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/add-page-alignment.patch > /tmp/add-page-alignment.patch
+    curl -L https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/ib_registration_cache.patch > /tmp/ib_registration_cache.patch
```

Comment:
Could just do `curl -LO` to download them to a file of the same name in the current working directory.

```suggestion
    curl -LO https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/add-page-alignment.patch
    curl -LO https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/ib_registration_cache.patch
```